homeassistant/components/zha/core/channels/base.py

```diff
@@ -155,13 +155,13 @@ def async_send_signal(self, signal: str, *args: Any) -> None:
 
     async def configure_reporting(
         self,
-        attr,
+        attr: Union[int, str]
```

Comment:
```suggestion
        attr: Union[int, str],
```
sorry, missed the comma